src/RuleSet.php

```diff
@@ -53,6 +53,7 @@
         ),
         '@Symfony' => array(
             '@PSR2' => true,
+            'array_syntax' => array('syntax' => 'long'),
```

Comment:
I was thinking about it. Symfony keeps that rule only for framework project to not have merging hell with old branches that supported 5.3.3.